presto-docs/src/main/sphinx/admin/verifier.rst

```diff
@@ -286,6 +288,15 @@ Name                                        Description
 ``running-mode``                            Set to ``query-bank`` to make the Verifier run in ``query-bank`` mode. Supports
                                             ``query-bank`` and ``control-test``. Defaults to ``control-test``.
 ``save-snapshot``                           Set to ``true`` to save checksums to ``mysql`` database.
+``nondeterministic-function-substitutes``   Specification of substituting nondeterministic functions with deterministic functions,
+                                            in the format of ```/foo_1(_,c_1)/bar_1(c_1)/,/foo_2(c_0,c_1)/bar_2(c_1,c_0)/,...``,
+                                            where ``foo_1`` would be substituted by ``bar_1`` with arguments applied
+                                            to the corresponding positions.
+                                            For example, ```/approx_percentile(x,_)/avg(x)/,/arbitrary(z)/min(z)/```. Make sure
```

Comment:
```suggestion
                                            For example, ```/approx_percentile(x,_)/avg(x)/,/arbitrary(z)/min(z)/```. 
                                            
                                            Make sure
```

Consider a paragraph break before "Make sure" to help the reader find the importance of this advice/

Reply:
Thanks @steveburnett for the suggestion! Revised

Reply:
> Thanks @steveburnett for the suggestion! Revised

You're welcome! I verify that I find the revision is present in a new local build of the docs.